engine/apps/mobile_app/models.py

```diff
@@ -10,6 +10,7 @@
 
 from apps.auth_token import constants, crypto
 from apps.auth_token.models import BaseAuthToken
+from apps.mobile_app.tasks import MessageType, Platform
```

Comment:
_nit_ maybe create a new file, `apps/mobile_app/types.py`, since these types aren't specific to the celery tasks

Reply:
Good call! Addressed in https://github.com/grafana/oncall/pull/2759/commits/ac895e6bc1521ec92cf9cf7ffde05a4161c5cb5c